src/parseArgs.ts

```diff
@@ -23,18 +23,9 @@ export const parseArgs = (): Namespace => {
 		action: 'store_true',
 		help: 'print substrate-api-sidecar version',
 	});
-	parser.add_argument('-p', '--prometheus', {
+	parser.add_argument('-mq', '--metrics_queryparams', {
```

Comment:
Maybe we can move also the option of query params in the config? That way all the configuration for metrics is in one place?

Reply:
Yep I agree with this!